### DISCUSSION TITLE
Need help on installing podman via installers (.pkg) in mac

### DISCUSSION THREAD

**Discussion by mohammed-muddassir** [containers/podman#23477] [General]
- I have been testing podman for a month and I'm going to suggest it to my colleagues. 
- Since there is a suggestion to not use brew to install podman. I'm trying to make a script to install podman with [cli installer](https://github.com/containers/podman/releases/download/v5.2.0-rc3/podman-installer-macos-universal.pkg)

- I dont need a user action by double clicking the .pkg file for installation after downloading
- Is there a way to do it via script in terminal itself?

```
# Download the .pkg file
curl -O https://github.com/containers/podman/releases/download/v5.2.0-rc3/podman-installer-macos-universal.pkg

# Install the .pkg file
sudo installer -pkg yourfile.pkg -target /
```

- Came across the above script online, 
I haven't manually installed a .pkg file in terminal before
Can someone help me with the script so that I can install podman via terminal without brew?
Thanks in advance.

**Comment by Luap99** [comment]
I tried that command and it worked without issues of me.

**Reply by mohammed-muddassir** [reply]
Thanks!
